### ISSUE TITLE
Recent updates to the node-test-runner have broken the run-tests.sh script

### ISSUE THREAD

**Issue by harrysarson** [elm/core#1103]
To reproduce:

```bash
./tests/run-tests.sh
```

The error is:

```
Failed to run `elm-json solve`:

-- NO VALID PACKAGE VERSION ----------------------------------------------------

Because elm/core does not appear to exist and this project depends on elm/core
at any version, no valid set of package versions could be found.



# clear out the copy of elm-core fetched by the above and replace it
# with the local source code we want to actually test

VERSION_DIR="$(ls ${ELM_HOME}/0.19.1/packages/elm/core/)"
ls: cannot access '/home/harry/repo/elm/core/tests/.elm/0.19.1/packages/elm/core/': No such file or directory
```

Looks like the upgrade to `elm-test@0.19.1-revision5` (where we added elm-json amongst other things) caused this script to fail.

**Comment by harrysarson** [comment]
As the one who released the offending version of elm-test, sorry!

**Comment by jfmengels** [comment]
@harrysarson What is your opinion on what should be done? Is this an issue you are tracking in `node-test-runner`?

**Comment by harrysarson** [comment]
I put a PR up at #1104 which is one way to fix this. @mpizenberg had a good idea on discord for how to keep using node-test-runner to run tests but I got distracted and never looked at it properly.

Given that tests.sh used all sorts of hacks to get this working I am pretty confident that the fix should be here.

**Comment by mpizenberg** [comment]
Yep, here is the fix: https://github.com/elm/core/compare/master...mpizenberg:tests-alternative

Works with both elm-test and elm-test-rs, and I think is simpler than the current setup

**Comment by harrysarson** [comment]
#1104 is now ready (from my end) to land. It is the minimal PR that fixes `./tests/run-tests.sh`. A follow up PR to migrate the CI to github actions (or to fix travis so that the status appears on PRs) would be nice but I decided to keep #1104 as simple as possible in the hope that is can be merged with minimum friction and unblock other PRs here. :)
